diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 536c404c2e..471091bc3c 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .38
-LINUX_KERNEL_HASH-6.12.38 = f035fa8d83d59f793c76b23567b130cc42118f10696815fed03c16bb15670fcc
+LINUX_VERSION-6.12 = .39
+LINUX_KERNEL_HASH-6.12.39 = e21ad84bae75104e15a40cc4d6c4bc20b3cd93bbe59de8984ec1b84c47eea0c3
